docs/composables/useTheme.md

```
@@ -96,3 +112,17 @@ export default {
+## Operation Configuration
+
+| Function             | Description                                        | Default Value    | Allowed Values                  |
+|----------------------|----------------------------------------------------|------------------|---------------------------------|
+| `getOperationBadges` | Gets the current operation badges.                 | `['deprecated']` | `['deprecated', 'operationId']` |
+| `setOperationBadges` | Sets the operation badges. The order is respected. | `['deprecated']` | `['deprecated', 'operationId']` |
+
+## I18n Configuration
+
+| Function        | Description                          | Default Value                                               | Allowed Values  |
+|-----------------|--------------------------------------|-------------------------------------------------------------|-----------------|
+| `getI18nConfig` | Gets the current i18n configuration. | `{ locale: 'en', fallbackLocale: 'en', messages: locales }` | `{ locale: 'es' | 'en', fallbackLocale: 'es' | 'en', messages: Record<'es' | 'en', Record<string, string>> }` |
```

Markdownlint

Expected: 4; Actual: 7; Too many cells, extra data will be missing
Table column count

(MD056, table-column-count)